build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,68 @@
+""" Script to create Sagemaker TensorFlow Docker images
+
+    Usage:
+        python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
+"""
+import argparse
+import glob
+import os
+import shutil
+import subprocess
+import sys
+
+## GLOBALS
+DOCKER = 'docker' # Stores which type of DOCKER to use, nvidia-docker or docker
```

Comment:
Are these needed since they're set in main?

If so, can you default PATH_TO_SCRIPT as well?

Reply:
I just have the GLOBALS for readability. I can remove that if that does not make sense.